go.mod

```diff
@@ -40,21 +40,22 @@ require (
 	golang.org/x/text v0.21.0
 	gomodules.xyz/jsonpatch/v2 v2.4.0
 	google.golang.org/grpc v1.65.1
-	k8s.io/api v0.31.4
-	k8s.io/apiextensions-apiserver v0.31.4
-	k8s.io/apimachinery v0.31.4
-	k8s.io/apiserver v0.31.4
-	k8s.io/client-go v0.31.4
+	k8s.io/api v0.32.0
+	k8s.io/apiextensions-apiserver v0.32.0
+	k8s.io/apimachinery v0.32.0
+	k8s.io/apiserver v0.32.0
```

Comment:
🤔 there's already v0.32.1? (ok to keep, I guess dependabot will bump)

Reply:
CR didn't bump ahead of the release (v0.32.1 was released 18h ago :D)

Bumped now